src/commands/starter/starter.command.ts

```diff
@@ -0,0 +1,121 @@
+        if (whoTag !== undefined ) {
+            // load in those saucy recources
+            const kitEmbed = new RichEmbed ({
+                title: '__**r/GM Resource Pack**__',
+                description: '**The following is a set of useful sources for learning GML, and other Gamemaker development skills.**',
+                thumbnail: {  url: 'https://www.yoyogames.com/images/gms2_logo_512.png'  },
+                fields: [
+                  {
+                    name: '**GML Written Articles:**',
+                    value: '[GMS1.x Documentation](http://docs.yoyogames.com/) \
+                    \n [GMS2 Documentation](http://docs2.yoyogames.com/) \
+                    \n Detailed Shader Basics: [Xor Shader Tutorials](https://xorshaders.weebly.com/) \
+                    \n Assorted Gamemaker Articles: [Amazon GM Recources](http://m.amazonappservices.com/GameMakerResources)'
+                  },
+
+                  {
+                    name: '**GML Video Tutorials and Channels:**',
+                    value: '[Pixelated Pope](https://www.youtube.com/user/PixelatedPope) \
+                    \n [Shaun Spalding](https://www.youtube.com/user/999Greyfox) \
+                    \n [HeartBeast](https://www.youtube.com/user/uheartbeast) \
+                    \n [Freindly Cosmonaut](https://www.youtube.com/channel/UCKCKHxkH8zqV9ltWZw0JFig) \
+                    \n [Very useful bite sized videos on specific topics](https://www.youtube.com/channel/UCcYKLm0EwyWkfTA6sMn5W7g) \
+                    \n [Official YYG Youtube](https://www.youtube.com/user/yoyogamesltd)'
+                  },
+
+                  {
+                    name: '**Advanced GML Topics:**',
+                    value: '[Git based source control with GMS2](https://www.youtube.com/watch?v=6zj86KN8Vco) \
+                    \n [Debug Mode Basics](https://www.youtube.com/watch?v=iJH_uTq9iOQ)'
+                  },
+
+                  {
+                    name: '**Basic Art Skill Articles:**',
+                    value: '[Guide to Being A Respectful Critic](http://tiny.cc/critique) \
+                    \n [Basic Guide to Pixelart](http://pixeljoint.com/forum/forum_posts.asp?TID=11299)'
+                  },
+
+                  {
+                    name: '**Art Video Recources and Channels:**',
+                    value: '[MortMort](https://www.youtube.com/user/atMNRArt) \
+                    \n [Guide on Choosing the Right Canvas Size](https://www.youtube.com/watch?v=AXb-VBZTKDA)'
+                  },
+
+                  {
+                    name: '**Pixelart Refrence Sites:**',
+                    value: '[Miniboss](http://blog.studiominiboss.com/pixelart) \
+                    \n [The Spriters Recource](https://www.spriters-resource.com/) \n'
+                  },
+
+                  {
+                    name: '**General Pixelart Tips:**',
+                    value: '-Start at a low resolution \
+                    \n -Use a pallete \
```

Comment:
I'd ask what others think about this, but I'd rather keep all of this to just links. This is good advice, but not necessarily the place.